docs/blog/2024-04-19-manage-your-gradle.md

```diff
@@ -12,7 +12,7 @@ We are very excited to announce our support for Gradle with our new plugin: `@nx
 
 The Nx Gradle plugin registers Gradle projects in your Nx workspace. It allows Gradle tasks to be run through Nx. Nx effortlessly makes your [CI faster](/ci/intro/ci-with-nx).
 
-> **Note:** this plugin is currently experimental.
+> **Update:** Learn how to import your existing Gradle projects into Nx in [our latest blog post](/blog/nx-import).
```

Comment:
I don't think we should use latest blog post here? It becomes out of date too quickly

```suggestion
> **Update:** Learn how to import your existing Gradle projects into Nx in [our recent blog post](/blog/nx-import).
```